src/main/java/nl/mtvehicles/core/movement/VehicleMovement.java

```diff
@@ -207,7 +207,7 @@ public void vehicleMovement(Player player, Object packet) {
             }
 
             updateStand();
-            if (!vehicleType.canFly()) blockCheck();
+            if (!vehicleType.isHelicopter()) blockCheck();
```

Comment:
Does this intentionally remove airplanes? (was there a bug related to their movement?)